src/client/components/pages/entities/work-table.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2019  Akhilesh Kumar (@akhilesh26)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import * as bootstrap from 'react-bootstrap';
+import * as entityHelper from '../../../helpers/entity';
+
+import Icon from 'react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+const {Button, Table} = bootstrap;
+
+const {getEntityDisambiguation} = entityHelper;
+
+function WorkTableRow({work}) {
+	const name = work.defaultAlias ? work.defaultAlias.name : '(unnamed)';
+	const disambiguation = getEntityDisambiguation(work);
+	const workType = work.workType ? work.workType.label : '?';
+	const workBBID = work.bbid;
+	return (
+		<tr>
+			<td>
+				<a href={`/Work/${workBBID}`}>{name}</a>
+				{disambiguation}
+			</td>
+			<td>{workType}</td>
+		</tr>
+	);
+}
+WorkTableRow.displayName = 'WorkTableRow';
+WorkTableRow.propTypes = {
+	work: PropTypes.object.isRequired
+};
+
+function WorkTable({entity}) {
+	return (
+		<div>
+			<h2>Works</h2>
+			<Table striped>
```

Comment:
If there are no works to be listed, I would replace the table with a more explicit call to action, like the one in the [search results|https://github.com/bookbrainz/bookbrainz-site/blob/master/src/client/components/pages/parts/search-results.js#L44], and perhaps a larger button